mlfinlab/codependence/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""
+Varoius codependence measure: mutual info, distance correlations, variation of information
+"""
+
+from mlfinlab.codependence.correlation import angular_distance, absolute_angular_distance, squared_angular_distance, \
```

Comment:
Uses braces for imports -> `from mlfinlab.codependence.information import (get_mutual_info, get_optimal_number_of_bins, variation_of_information_score)`